src/lib.rs

```diff
@@ -675,6 +720,41 @@ impl<K: KVStore + Sync + Send + 'static> Node<K> {
 		self.runtime.read().unwrap().is_some()
 	}
 
+	async fn create_channel_from_pj(
+		pj_req: PayjoinRequest, _payjoin_executer: Arc<PayjoinExecuter<K>>,
+	) {
+		let psbt = pj_req.clone().original_psbt();
+		let pj_response = PayjoinResponse::new(psbt);
+		// Send OpenChannel message
```

Comment:
This is where the channel negotiations should happen and the `queue` the response back to the http handler